.github/workflows/ci.yml

```diff
@@ -284,6 +284,21 @@ jobs:
       run: ./run.sh DEFAULT
       env:
         DD_API_KEY: ${{ secrets.DD_API_KEY }}
+
+
+
+
+    - name: Run CROSSED_INTEGRATIONS scenario
+      if: steps.build.outcome == 'success'
+      run: |
+        ./utils/build/build_tracer_buddies.sh  # TODO: publish those images to docker hub
```

Comment:
For now, I built the images locally via `docker build -t system_tests/python_buddy -f utils/build/docker/python/flask-poc.Dockerfile .`